docs/learn-faq.md

```diff
@@ -30,11 +30,11 @@ be quite fast. Two or three seconds may not be unreasonable after optimizations.
 
 ### What is the token inflation rate?
 
-This is difficult to calculate precisely, since the inflation rate will vary based on what percentage of the DOTs are staked. For Kusama, inflation is estimated to be around 10 - 20% for the first year.
+This is difficult to calculate precisely, since the inflation rate will vary based on what percentage of the DOTs are staked. For both Kusama and Polkadot, inflation is estimated to be around 10 - 20% for the first year.  Inflation on testnets and parachains will vary more dramatically.
```

Comment:
```suggestion
This is difficult to calculate precisely, since the [inflation rate will vary](learn-staking#inflation) based on what percentage of the DOTs are staked. For both Kusama and Polkadot, inflation is estimated to be around 10 - 20% for the first year.  Inflation on testnets and parachains will vary more dramatically.
```